frontend/src/metabase/static-viz/index.js

```diff
@@ -1,3 +1,5 @@
+import "fast-text-encoding";
```

Comment:
what is this?

Reply:
This is a shim for static viz. The JS environment that static viz doesn't include `TextEncocer`, which React 18 assumes is available